tools/serverpod_cli/lib/src/util/model_helper.dart

```diff
@@ -87,7 +87,7 @@ class ModelHelper {
   ) async {
     List<FileSystemEntity> modelSourceFileList = [];
 
-    var dir = Directory('/${joinAll(absolutePathParts)}');
+    var dir = Directory('$separator${joinAll(absolutePathParts)}');
```

Comment:
I see that you fix the slash, but maybe the path needs to be built in a different way. Isn't the directory always absolut if you give a path?